holoviews/plotting/bokeh/callbacks.py

```diff
@@ -92,19 +104,41 @@ class Callback(object):
     attributes = {}
 
     js_callback = """
+        function unique_events(events) {{
+            var unique = [];
+            var unique_events = [];
+            for (var i=0; i<events.length; i++) {{
+                [event, data] = comm_state.events[i];
+                if (!unique_events.includes(event)) {{
+                    unique.unshift(data);
+                    unique_events.push(event);
+                }}
+            }}
+            return unique;
+        }}
+
+        function process_events(comm_state) {{
+            var events = unique_events(comm_state.events);
```

Comment:
Fair enough - I was just trying to understand the code out of my own curiosity and thought I might as well make some comments while I'm at it.